packages/react-native-reanimated/Common/cpp/reanimated/Tools/FeatureFlags.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include <string>
+#include <unordered_map>
+
+namespace reanimated {
+
+class StaticFeatureFlags {
+ public:
+#ifdef REANIMATED_FEATURE_FLAGS
+
+#define STRINGIFY(x) #x
+#define TOSTRING(x) STRINGIFY(x)
+#define REANIMATED_FEATURE_FLAGS_STRING TOSTRING(REANIMATED_FEATURE_FLAGS)
+
+  static constexpr bool getFlag(const char *key) {
+    const std::string keyStr = key;
+    std::string featureFlags = REANIMATED_FEATURE_FLAGS_STRING;
+    if (featureFlags.find(key) == std::string::npos) {
```

Comment:
I don't think this will work properly if we have two flags where one is contained within another, e.g. `cde` and `abcdefg`. We should look for `[cde:` instead of only `cde`.